src/main/java/no/ndla/taxonomy/domain/Node.java

```diff
@@ -113,6 +129,88 @@ public Node(Node node, boolean keepPublicId) {
         setName(node.getName());
     }
 
+    public Optional<Grade> getQualityEvaluationGrade() {
+        return Optional.ofNullable(qualityEvaluation);
+    }
+
+    public Optional<String> getQualityEvaluationComment() {
+        return Optional.ofNullable(qualityEvaluationComment);
+    }
+
+    public Optional<String> getQualityEvaluationNote() {
+        return Optional.ofNullable(qualityEvaluationComment);
+    }
+
+    public Optional<GradeAverage> getChildQualityEvaluationAverage() {
+        return Optional.ofNullable(this.childQualityEvaluationAverage)
+                .map(avg -> new GradeAverage(avg, this.childQualityEvaluationCount));
+    }
+
+    @Transactional
```

Comment:
Meiner du ikkje treng å ha dette med i domeneklassen siden funksjonen uansett kalles fra en service som er transactional. Du kunne evt lagt på `@Transactional(propagation = Propagation.MANDATORY)` for å seie at kall på funksjonen krever en aktiv transasjon men ikkje oppretter ny.